vaadin-dialog-flow-parent/vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -106,10 +106,11 @@ public Dialog() {
         container.getClassList().add("draggable-leaf-only");
         container.getStyle().set(ElementConstants.STYLE_WIDTH, "100%");
         container.getStyle().set(ElementConstants.STYLE_HEIGHT, "100%");
+        container.getStyle().set("display", "inline-block");
```

Comment:
This fix was originally added in https://github.com/vaadin/vaadin-dialog-flow/pull/195 and then removed in https://github.com/vaadin/vaadin-dialog-flow/pull/232 due to the regression it caused. Since `<flow-component-renderer>` is now removed, I reverted back to the original fix.

Tested the change and both of the previous Dialog scrollbar issues are covered:
- https://github.com/vaadin/vaadin-dialog-flow/issues/185
* https://github.com/vaadin/vaadin-dialog-flow/issues/225

Consider this a temporary fix, since the remaining unnecessary `<div>` wrapper will be completely removed by a separate follow-up PR.